### PR TITLE
Add a link to delete records

### PR DESCRIPTION
**Pull request by magopian**

### CODE REVIEW

File: src/Model.elm [hunk 1]
```diff
@@ -98,6 +102,20 @@ update msg model =
...
 
+        DeleteRecord recordId ->
+            ( model, deleteRecord recordId )
+
+        DeleteRecordSucceed { data } ->
+            ( { model
+                | records = removeRecordFromList data model.records
+                , error = False
+              }
+            , Cmd.none
```
Comment (n1k0) [inline src/Model.elm:120]: I'm wondering if we shouldn't trigger a full refresh of the list instead/along of just removing the entry from our local one?

**Review by n1k0 — COMMENTED** [review]
Looks good!

File: src/Model.elm (hunk 1 above)
Comment (magopian) [reply src/Model.elm:120]: Excellent question. We should take some time to see if we want to "experiment" with optimistic updates or not, and if we want to try optimizing (removing an item from the list) or not (refreshing the whole list).

Comment (n1k0) [reply src/Model.elm:120]: We should probably land it as is and iterate later :)

**Comment by n1k0** [comment]
If you rebase this, I land it :)

**Comment by magopian** [comment]
done ;) (as discussed above, it's just a proof of concept, we might want to discuss optimistic updates, and either ditch them, or "combine" them with a full refresh afterwards. There's some interesting challenges there, like "how do I show the user his input is being processed, how do I show him it's "only" an optimistic update...")

**Comment by n1k0** [comment]
Yes I know, but we can always revisit the strategy after the actual feature is landed, especially as it works and so we're gonna grab actual feedback while using it :)

**Pull request merged**